src/language-js/utils.js

```diff
@@ -115,7 +134,7 @@ function getLeftSidePathName(path, node) {
   if (node.expression) {
     return ["expression"];
   }
-  throw new Error("Unexpected node has no left side", node);
+  throw new Error("Unexpected node has no left side.");
```

Comment:
I just discovered this change and am a little confused by it.

I had thought it would be nice if the exception would show the `node` object that is malformed if we ever get to this point.

Reply:
`Error` constructor only take one  argument.